server/validators/dateIsAfterExpectedReleaseDate.ts

```diff
@@ -2,19 +2,20 @@ import { registerDecorator, ValidationArguments, ValidationOptions } from 'class
 import moment, { Moment } from 'moment'
 import _ from 'lodash'
 import type SimpleDate from '../routes/creatingLicences/types/date'
+import type { DateString } from '../routes/creatingLicences/types/dateString'
 import UkBankHolidayFeedService from '../services/ukBankHolidayFeedService'
 
 export default function DateIsAfterExpectedReleaseDate(validationOptions?: ValidationOptions) {
   const bankHolidayService = new UkBankHolidayFeedService()
 
-  const dateIsAfterExpectedReleaseDate = async (date: SimpleDate, { object }: ValidationArguments) => {
+  const dateIsAfterExpectedReleaseDate = async (date: SimpleDate | DateString, { object }: ValidationArguments) => {
     const bankHolidays = await bankHolidayService.getEnglishAndWelshHolidays()
 
     const getWorkingDayOnOrBefore = (d: Moment): Moment => {
       return bankHolidays.isBankHolidayOrWeekend(d) ? getWorkingDayOnOrBefore(d.subtract(1, 'day')) : d
     }
 
-    const dateAsMoment = date.toMoment()
+    const dateAsMoment = typeof date === 'string' ? moment(date, 'DD/MM/YYYY') : date.toMoment()
```

Comment:
would be tempted to wrap the string in a DateString class, with a `toMoment`method and `@Expose` on the string.
Then these could work on things with `toMoment`s on it - rather than do extra parsing here.

Reply:
sorted in a follow up PR: https://github.com/ministryofjustice/create-and-vary-a-licence/pull/751